src/doc/languagespec.tex

```diff
@@ -182,7 +182,7 @@
 \smallskip
 \spc \includegraphics[width=0.85in]{Figures/CC-BY.png}
 \spc http://creativecommons.org/licenses/by/4.0/
- \bigskip 
```

Comment:
Can you please revert languagespec.tex? It's just making whitespace changes all throughout.

Reply:
Will do - it looks like my editor is trimming whitespace on the whole file instead of on edited lines.